### PR TITLE
Feat/update dse

### PR DESCRIPTION
**Pull request by clementbiron**
@martinratinaud I am not satisfied with the tsconfig.json modification to fix `Could not find declaration file` error
Maybe a better way to do it ? 
Look like a good reading about it: https://blog.atomist.com/declaration-file-fix/

### CODE REVIEW

**Comment by martinratinaud** [comment]
Sure

I usually do Best Fix #7: Get the type declarations from @types
and if  not Kind-of-terrible Fix #4: Allow this one JS module import

What would you do ?

File: tsconfig.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
     "strict": true,
     "target": "esnext",
 
-    //ttps://stackoverflow.com/questions/50555859/typescript-could-not-find-a-declaration-file-for-module-react-cards
+    // Because importing npm modules dsfr trow an error "Typescript: Could not find a declaration file for module"
+    // -> We have to disable warning
+    // Read  ttps://stackoverflow.com/questions/50555859/typescript-could-not-find-a-declaration-file-for-module-react-cards
```
Comment (martinratinaud) [inline tsconfig.json:24]: Indeed this is clearly really bad and should be removed

**Review by martinratinaud — APPROVED** [review]
Seems fine to me but not tested on my local

File: tsconfig.json (hunk 1 above)
Comment (clementbiron) [reply tsconfig.json]: ok i dit it

**Comment by clementbiron** [comment]
> Sure
> 
> I usually do Best Fix #7: Get the type declarations from @types
> and if not Kind-of-terrible Fix #4: Allow this one JS module import
> 
> What would you do ?

I used fix number 4.
Look good to you ?

**Comment by martinratinaud** [comment]
very good

maybe file an issue in their repo to generate their typescript types also :-)

**Comment by clementbiron** [comment]
`@gouvfr/dsfr` has not yet public repo 😭

**Pull request merged**